simulator/CMakeLists.txt

```diff
@@ -94,3 +94,6 @@ target_link_libraries(${PROJECT_NAME} mipt-mips-src)
 
 # Tests
 include(${CMAKE_CURRENT_LIST_DIR}/tests/CMakeLists.txt)
+
+# GDB libsim.a
+include(gdb/mips-sim-gdb.cmake)
```

Comment:
Please add `interface` directory between `simulator` and `gdb`.
New interfaces will be placed here.